### PR TITLE
changing based on new information

### PR DESCRIPTION
**Pull request by anwara-ms**
changing based on new information

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Information barriers (IB) are policies that an admin can configure to prevent in
 
 > [!NOTE]
 > - Information barrier groups cannot be created across tenants.
-> - Using bots to add users is not supported in version 1.
+> - Using bots/AAD Apps/some APIs to add users is not supported in version 1.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:27]:
```suggestion
> - Using bots, AAD apps, and some APIs to add users is not supported in version 1.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by anwara-ms** [comment]
when is it going to go live?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@LauraWi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LauraWi** [comment]
@Christopher J Fox (OFFICE)<mailto:chrfox@exchange.microsoft.com> -- Since this is Information Barrers, can you take a look? Thanks!
________________________________

**Comment by LauraWi** [comment]
Assigned myself, merged.  Should be live in 24 -35 hrs

Christopher J. Fox
Senior Content Developer |  425 652 2423 | chrfox@microsoft.com<mailto:chrfox@microsoft.com>
I identify with these pronouns he, him, his

**Pull request merged**